packages/strapi-plugin-content-type-builder/admin/src/containers/FormModal/utils/createUid.js

```diff
@@ -1,6 +1,4 @@
-import slugify from '@sindresorhus/slugify';
-
-const nameToSlug = name => slugify(name, { separator: '-' });
+import { nameToSlug } from 'strapi-utils';
```

Comment:
Strapi-utils isn't meant to be used on the front-end :) Can you reset those changes :)